14_functions_lab/04.py

```diff
@@ -1,6 +1,11 @@
-"""
-Write a function that takes minlen and
-a list of words, and returns only the words
-longer than minlen
-"""
-
+"""
+"""
+
+def longer_than(length, *text):
+    for word in text:
+        if len(word) > length:
+            yield word

```

Comment:
Note that the task was to return a list. Yield works for printing but if we wanted for instance the length yield wouldn't be that great. The following code breaks in your program:

```
print len(longer_than(3, "tomer", "nir", "hello", "a"))
```